### DISCUSSION TITLE
SuperSlicer Feature: Reverse on odd

### DISCUSSION THREAD

**Discussion by MauriMo94** [OrcaSlicer/OrcaSlicer#1812] [Ideas]
This feature improves Overhangs a lot by reversing the extrusion direction on odd layers. It would be great to see in Orcaslicer, too :)![image](https://github.com/SoftFever/OrcaSlicer/assets/82610675/3e32235c-649f-4894-82cd-714b4b018c49)

**Comment by Schokobecher** [comment]
This is now available in the 1.8.0 Beta :)

> Change direction for perimeter extrusion at odd layers. by @Noisyfox in https://github.com/SoftFever/OrcaSlicer/pull/2413

**Reply by MauriMo94** [reply]
Awesome! This feature will improve overhang quality a lot 💪
